lib/src/client.dart

```diff
@@ -79,6 +83,7 @@ class NextCloudClient {
         NextCloudHttpClient.withoutLogin(
           language,
         ),
+        '', // We can leave this empty, because this client can't be use for requests at all
```

Comment:
We could introduce two interfaces, which are both implemented by the `NextcloudClient` and hiding the not available functionality in the respective modes of the `NextcloudClient`.

Reply:
But this nothing urrgend.